libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-details.component.ts

```diff
@@ -68,7 +68,7 @@ import { ElectricityMarketMeteringPointType } from '@energinet-datahub/dh/shared
   `,
   template: `
     <watt-card *transloco="let t; read: 'meteringPoint.overview.details'">
-      <div class="grid-wrapper">
+      <div class="grid-wrapper" [class.have-children]="true">
```

Comment:
nitpick: a slip? 🙃

Reply:
Yes, were started on a new issue, but forgot to stash the changes